### PR TITLE
Update Descriptions in Diffusion Model Training Notebook

### PR DESCRIPTION
**Pull request by dongyang0122**
### Description
Update descriptions in diffusion model training notebook.

### Checks

- [ ] Avoid including large-size files in the PR.
- [ ] Clean up long text outputs from code cells in the notebook.
- [ ] For security purposes, please check the contents and remove any sensitive info such as user names and private key.
- [ ] Ensure (1) hyperlinks and markdown anchors are working (2) use relative paths for tutorial repo files (3) put figure and graphs in the `./figure` folder
- [ ] Notebook runs automatically `./runner.sh -t <path to .ipynb file>`

### CODE REVIEW

File: generative/maisi/maisi_diff_unet_training_tutorial.ipynb [hunk 1]
```diff
@@ -22,15 +22,21 @@
    "id": "777b7dcb",
```
Comment (KumoLiu) [inline generative/maisi/maisi_diff_unet_training_tutorial.ipynb:22]: <p>Let's leave this on as "Setup environment" to be consistent with the other notebooks.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Project-MONAI/tutorials/pull/1761/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

**Review by KumoLiu — APPROVED** [review]
Thanks for the PR, overall looks good to me.

File: generative/maisi/maisi_diff_unet_training_tutorial.ipynb [hunk 2]
```diff
@@ -22,15 +22,21 @@
...
+    "In this notebook, we detail the procedure for training a 3D latent diffusion model to generate high-dimensional 3D medical images. Due to the potential for out-of-memory issues on most GPUs when generating large images (e.g., those with dimensions of 512 x 512 x 512 or greater), we have structured the training process into two primary steps: 1) generating image embeddings and 2) training 3D latent diffusion models. The subsequent sections will demonstrate the entire process using a simulated dataset."
    ]
   },
   {
    "cell_type": "markdown",
    "id": "c9ecfb90",
    "metadata": {},
    "source": [
-    "## Setup environment"
+    "## Step 0: Setup environment\n",
```
Comment (mingxin-zheng) [inline generative/maisi/maisi_diff_unet_training_tutorial.ipynb:37]: Hi @dongyang0122 , based on the the [contributing guideline](https://github.com/Project-MONAI/tutorials/blob/main/CONTRIBUTING.md), can we revert the 
```
## Step 0: Setup environment\n
```
Back to 
```
## Setup environment
```

A uniform style and format helped us to run automated scripts.

File: generative/maisi/maisi_diff_unet_training_tutorial.ipynb [hunk 3]
```diff
@@ -49,7 +55,7 @@
    "id": "d655b95c",
    "metadata": {},
    "source": [
-    "## Setup imports"
+    "### Setup imports"
```
Comment (mingxin-zheng) [inline generative/maisi/maisi_diff_unet_training_tutorial.ipynb:58]: Same here, can we revert the change 
```
### Setup imports
```
To 
```
## Setup imports
```
Thanks!

File: generative/maisi/maisi_diff_unet_training_tutorial.ipynb (hunk 1 above)
Comment (dongyang0122) [reply generative/maisi/maisi_diff_unet_training_tutorial.ipynb:22]: Thanks. @KumoLiu I made according changes. Please proceed.

File: generative/maisi/maisi_diff_unet_training_tutorial.ipynb (hunk 2 above)
Comment (dongyang0122) [reply generative/maisi/maisi_diff_unet_training_tutorial.ipynb]: Thanks. @mingxin-zheng I made according changes. Please proceed.

File: generative/maisi/maisi_diff_unet_training_tutorial.ipynb (hunk 3 above)
Comment (dongyang0122) [reply generative/maisi/maisi_diff_unet_training_tutorial.ipynb]: Thanks. @mingxin-zheng I made according changes. Please proceed.

**Review by mingxin-zheng — APPROVED** [review]

File: generative/maisi/maisi_diff_unet_training_tutorial.ipynb (hunk 2 above)
Comment (mingxin-zheng) [reply generative/maisi/maisi_diff_unet_training_tutorial.ipynb]: Thanks for the edits @dongyang0122 !
